recipe/meta.yaml

```diff
@@ -9,8 +10,10 @@ source:
   md5: a77c630fca481f4f533930d406bd9701
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and (vc<14 or win32)]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
```

Comment:
I don't think it's a good idea to do this without updating the version. How would downstream packages make sure they get this build number instead of the old one? At least, if we do this in the next version, downstream packages can do something like:
```yaml
requirements:
  build:
    - libmatio > 1.5.19
```
How about adding yourslef as a maintainer and adding this when the next version arrives?

Reply:
If downstream users want to absolutely make sure to have run dependency of libmatio, they can simply add directly in run section libmatio with the desired dependency? This modifications will still work fine for the vast majority of downstream users that will get the latest matio unless there are some strange solver problems.

Reply:
ok, I guess also other tests could also prevent a problem in downstream packages.